tests/unit/type/stream-cgroups.tcl

```diff
@@ -1300,6 +1300,18 @@ start_server {
         assert_equal [dict get $group entries-read] 1
         assert_equal [dict get $group lag] 1
 
+        # When all the entries are read, the lag is always 0.
+        r XREADGROUP GROUP mygroup alice STREAMS x >
+        set reply [r XINFO STREAM x FULL]
+        set group [lindex [dict get $reply groups] 0]
+        assert_equal [dict get $group lag] 0
```

Comment:
```suggestion
        set group [lindex [dict get $reply groups] 0]
        assert_equal [dict get $group entries-read] 5
        assert_equal [dict get $group lag] 0
```
At this time, cg->last_id == s->first_id, so we can cover it.
Although a similar check already exists below, it tests for cg->last_id > s->first_id.